bonnie-businessrules/src/main/java/com/cgi/bonnie/businessrules/user/User.java

```diff
@@ -10,11 +10,7 @@ public class User {
 
     Role role;
 
-    public User(long id, String name, Role role) {
-        this.id = id;
-        this.name = name;
-        this.role = role;
-    }
+    String password;
```

Comment:
please don't put this field into the User object. We don't want to deal with it here. It's enough if we have a field in AssemblyUser for the password.

Reply:
ok

Reply:
on login, it can be handled as a different parameter to authenticate, and later on we wont need it.